packages/sdk/src/gre/task.ts

```diff
@@ -0,0 +1,32 @@
+import { task } from 'hardhat/config'
+import { ActionType, ConfigurableTaskDefinition } from 'hardhat/types/runtime'
+
+function grePrefix(text: string): string {
+  return `[GRE] ${text}`
+}
+
+export function greTask(
+  name: string,
+  description?: string | undefined,
+  action?: ActionType<unknown> | undefined,
+): ConfigurableTaskDefinition {
+  return task(name, description, action)
+    .addOptionalParam('addressBook', grePrefix('Path to the address book file.'))
```

Comment:
I'm not following where the default values are now, or we don't have defaults anymore?

Reply:
ah, is it via hardhat config?